examples/react-client/minimal-react/src/components/App.tsx

```diff
@@ -14,9 +12,8 @@ const FISHJAM_URL = "ws://localhost:5002";
 export const App = () => {
   const [token, setToken] = useState("");
 
-  const connect = useConnect();
-  const disconnect = useDisconnect();
-  const status = useStatus();
+  const { joinRoom, leaveRoom, peerStatus } = useConnection();
```

Comment:
Maybe it makes sense. But this should be discussed. Also it would require to refactor a lot of code:
- Update all `PeerStatus` references (including `PeerStatus` type)
- Make same changes on mobile
- Update docs (that would be easies part).